libs/auth/src/dipped_smart_card_auth.ts

```diff
@@ -384,97 +384,93 @@ export class DippedSmartCardAuth implements DippedSmartCardAuthApi {
 
     switch (action.type) {
       case 'check_card_reader': {
-        const newAuthStatus = ((): DippedSmartCardAuthTypes.AuthStatus => {
```

Comment:
This commit is easiest to review using [?w=1](https://github.com/votingworks/vxsuite/pull/3135/commits/676ce525fb7f50a9a1b7b67e84cfe375b26262df?w=1)